modules/registrars/synergywholesaledomains/synergywholesaledomains.php

```diff
@@ -2287,7 +2288,7 @@ function synergywholesaledomains_ClientAreaCustomButtonArray(array $params)
         'Manage DNSSEC Records'     => 'manageDNSSEC',
     ];
 
-    if (substr($params['tld'], -3) == '.au') {
+    if (Str::endsWith($params['tld'], '.au')) {
```

Comment:
I think direct au domains just return 'au' and not '.au', for the other TLD's this works cause it would be 'com.au'

Can we double check what a direct au domain returns as the TLD?

Reply:
Correct, i had it changed to au while testing and never pushed the changes